src/main/java/org/sonarsource/sonarlint/ls/AnalysisManager.java

```diff
@@ -88,14 +90,19 @@
 
   private static final Logger LOG = Loggers.get(AnalysisManager.class);
 
+  private static final String SECURITY_REPOSITORY_HINT = "security";
```

Comment:
Note to myself, we should probably share the logic to determine if issues are taint vulnerabilities in the core...